code/modules/mob/living/carbon/human/species_types/shadowpeople.dm

```diff
@@ -218,6 +218,10 @@
 		playsound(src, 'sound/items/cig_snuff.ogg', 50, 1)
 	..()
 
+/obj/structure/glowshroom/lighteater_act(obj/item/light_eater/light_eater)
+	..()
+	acid_act()
```

Comment:
Shouldn't this only affect it if it has a positive light_power, so shadowshrooms aren't affected?

Reply:
shadowshroom items already get disintegrated